cfbs/masterfiles/generate_vcf_git_checkout.py

```diff
@@ -0,0 +1,101 @@
+import os
+import shutil
+import subprocess
+import sys
+
+from cfbs.utils import write_json
+from cfbs.masterfiles.analyze import (
+    finalize_vcf,
+    initialize_vcf,
+    versions_checksums_files,
+)
+
+DIR_PATH = "."
+"""The path of the working directory."""
+
+MPF_URL = "https://github.com/cfengine/masterfiles"
+MPF_PATH = os.path.join(DIR_PATH, "masterfiles")
+
+
+def check_required_command(command):
+    if not shutil.which(command):
+        print("`%s` was not found" % command)
+        sys.exit(1)
+
+
+def check_required_commands(commands):
+    for c in commands:
+        check_required_command(c)
```

Comment:
These two can probably be the same function. If the passed argument is an interable, then it can recursively call itself for each element in the iterable.